### DISCUSSION TITLE
Best Zigbee adapter available in US?

### DISCUSSION THREAD

**Discussion by t0ny-peng** [Koenkk/zigbee2mqtt#7823] [General]
Hi guys. I'm living in US and want to know what's the best one you can get in US. I saw `zig-a-zig-ah` is the recommend model but can't find it on Amazon, eBay or Aliexpress. Seems that it's only sold in Europe?

Thanks in advance.

**Comment by DubhAd** [comment]
Try [Tube's store](https://www.tubeszb.com/shop/coordinators/2) - though he's currently out of stock.

**Comment by gio-dot** [comment]
Hi, take a look here: Z-Bee duo has the particularity that can be used in 3 modes. Can be useful when you have to place it away from your Homeassistant gateway. https://github.com/Gio-dot/Z-Bee-Duo

**Reply by t0ny-peng** [reply]
@Gio-dot Thanks..But it's shipped from Italy and out of stock.

**Reply by gio-dot** [reply]
It will be again in stock in about 2 weeks. I shipped many times in US.

**Reply by t0ny-peng** [reply]
@Gio-dot Cool. Do you have a reseller partner in US? Or are you interested in finding one?

**Comment by mercenaruss** [comment]
Check out also zig-star.com
